.github/workflows/localize.yml

```diff
@@ -26,12 +26,6 @@ jobs:
         TDBUILD_AAD_APPLICATION_CLIENT_SECRET: ${{ secrets.TDBUILD_AAD_APPLICATION_CLIENT_SECRET }}
       run: scripts/localize.sh
     - name: Create Pull Request
-      uses: peter-evans/create-pull-request@v3.5.0
-      with:
-        commit-message: Latest localized resource files from Touchdown Build
-        title: '[Localization] Localized Resource Files'
-        body: 'Latest localized resource files from Touchdown Build'
-        branch: tdbuild
-        labels: Automated PR
-        branch-suffix: timestamp
-    
+      run: git checkout -b 'tdbuild`date +%s`'; git push --set-upstream origin huwilkes/PRActions`date +%s`; gh pr create -B main -H tdbuild`date +%s` --title '[Localization] Localized Resource Files' --body  'Latest localized resource files from Touchdown Build' -l 'Automated PR'
```

Comment:
Do you intend to be referencing `huwilkes/PRActions` branch?

Reply:
Also wondering if this script is long enough that it should live in a dedicated file under `scripts/` instead of being inline here?

Reply:
I sure didn't meant to leave that in there. Fixed, thanks!

Reply:
I think it's fine inline for now, we can move it to a script if we even need it with the app.

Reply:
Though I was slightly worried about the output from date changing while it ran, it seems to be fine. Probably good to move it out, though.

Reply:
script should work according to https://github.com/huwilkes/fluentui-apple/pull/7